src/files.cpp

```diff
@@ -813,3 +908,31 @@ std::list<std::string> directoryContents(const char* directory)
 
 	return list;
 }
+
+std::vector<std::string> getLinesFromDataFile(std::string filename)
+{
+	std::vector<std::string> lines;
+	FILE *file = fopen(filename.c_str(), "r");
+	if ( !file )
+	{
+		printlog("Error: Failed to open file \"%s\": ", filename.c_str(), strerror(errno));
+		return lines;
+	}
+	size_t line_size = 0, line_length;
+	char *line = NULL;
+	while ( (line_length = getline(&line, &line_size, file)) != -1 )
```

Comment:
Where are you getting `getline()` from? Windows is complaining it's not defined.

Reply:
>       Both getline() and getdelim() were originally GNU extensions.  They
>       were standardized in POSIX.1-2008.

Bugger. I'll have a look at replacing it soon (today or tomorrow probably).